static/src/js/components/Facets/FacetsList.js

```diff
@@ -11,22 +17,56 @@ const FacetsList = (props) => {
     facets,
     facetCategory,
     liftSelectedFacets,
-    changeHandler
+    changeHandler,
+    sortBy,
+    variation
   } = props
 
-  let orderedFacets = []
+  // Start by creating arrays to sort lifted and non-lifted facets
```

Comment:
There is a lot of logic in this component, could it be stripped out into a util function(s) to make this file easier to read?